src/main.c

```diff
@@ -589,6 +589,12 @@ static void do_arg()
   } else if (cliflags & 1) {
     show_ver();
     exit(0);
+  } else if ((cliflags & 8) || (cliflags & 4)) {
+    if (!(cliflags & 16)) {
```

Comment:
Combine these to `} else if (!(cliflags & 16) && (cliflags & 8 || cliflags & 4)) {`, otherwise the WARNING of multiple confs won't be shown when using `./eggdrop -nt bot.conf etc.conf` (and `-nc`).